vNext/test/ServiceBus.Tests/ServiceBus.Tests.csproj

```diff
@@ -23,7 +23,7 @@
     <Reference Include="Microsoft.CSharp" />
   </ItemGroup>
   <ItemGroup>
-    <PackageReference Include="FluentAssertions" Version="4.0.0" />
+    <PackageReference Include="FluentAssertions" Version="4.18.0" />
```

Comment:
I noticed this dependency once... are we even using it? If it's 1 codefile that uses it, I would prefer to just get rid of it.
I liked and used FluentAssertions, but I prefer consistency... Not necessarily to be done in this PR